### ISSUE TITLE
Use uncalibrated energy as waveform max instead of actual max for timepoints

### ISSUE THREAD

**Issue by hervasa2** [legend-exp/LegendDSP.jl#43]
We should take care to get all PSD parameters from normalized waveforms. Here the normalization factor (`E`) should be the same across the processing chain (`A/E`, `QDrift/E`, ...) and equal to a filtered maximum of the waveform in ADC.  
Therefore in: 
https://github.com/legend-exp/LegendDSP.jl/blob/acb4c5679a80bab70a602d3a0398082b170044c8/src/dsp_icpc.jl#L138-L142
`wvf_max` should be replaced by `e_cusp` or `e_trap`

**Comment by theHenks** [comment]
Agreed, that should be an easy fix. I will include a new `metadata` parameter which controls which energy estimator will be used for that. 
I don't think it makes a huge difference right now but you are right. 
For the current processing, I would go with `e_cusp` then since this is at the moment our default energy estimator.
